[2025-02-21T07:38:51.838+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T05:26:00+00:00 [queued]>
[2025-02-21T07:38:51.910+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T05:26:00+00:00 [queued]>
[2025-02-21T07:38:51.914+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-02-21T07:38:52.131+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_task> on 2025-01-01 05:26:00+00:00
[2025-02-21T07:38:52.279+0000] {standard_task_runner.py:60} INFO - Started process 3260 to run task
[2025-02-21T07:38:52.328+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'optum_etl_dag1', 'load_task', 'scheduled__2025-01-01T05:26:00+00:00', '--job-id', '737', '--raw', '--subdir', 'DAGS_FOLDER/optum_etl_dag1.py', '--cfg-path', '/tmp/tmpj9n5yho4']
[2025-02-21T07:38:52.342+0000] {standard_task_runner.py:88} INFO - Job 737: Subtask load_task
[2025-02-21T07:38:53.205+0000] {task_command.py:423} INFO - Running <TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T05:26:00+00:00 [running]> on host 491940227afd
[2025-02-21T07:38:54.030+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='optum_etl_dag1' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T05:26:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T05:26:00+00:00'
[2025-02-21T07:38:54.269+0000] {logging_mixin.py:188} INFO - Loaded data: {'name': 'AIRFLOW', 'type': 'ORCHESTRATION'}
[2025-02-21T07:38:54.285+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-02-21T07:38:54.525+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=optum_etl_dag1, task_id=load_task, execution_date=20250101T052600, start_date=20250221T073851, end_date=20250221T073854
[2025-02-21T07:38:54.767+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-21T07:38:54.906+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
